### DISCUSSION TITLE
How can we get the mouse/touch position in a container?

### DISCUSSION THREAD

**Discussion by LeviSTA** [pixijs/pixijs#9287] [Q&A] [answered]
Hi there,

I'm trying to get the mouse/touch position where we clicked in the container.

I appreciate any help you can provide.

**Comment by SuperSodaSea** [comment, marked as answer]
Try [`getLocalPosition`](https://pixijs.download/dev/docs/PIXI.FederatedMouseEvent.html#getLocalPosition):

```js
container.on('mousemove', e => {
    console.log(e.getLocalPosition(container));
});
```

**Reply by LeviSTA** [reply]
> Try [`getLocalPosition`](https://pixijs.download/dev/docs/PIXI.FederatedMouseEvent.html#getLocalPosition):
> 
> ```js
> container.on('mousemove', e => {
>     console.log(e.getLocalPosition(container));
> });
> ```

Sadly this doesn't execute anything in the console.

**Reply by SuperSodaSea** [reply]
You need to set `container.interactive = true` (Before PixiJS 7.2.0) or `container.eventMode = 'static'` (After PixiJS 7.2.0).
Example here: <https://www.pixiplayground.com/#/edit/QsPRWhDZg2NKc1XcsaAVy>

**Reply by LeviSTA** [reply]
Didn't work either. It doesn't execute the log. It might be because when I log the container's bounds, all are 0. 

But this is weird, as I don't necessarily change anything about this.

**Reply by LeviSTA** [reply]
Nvm he does appear to have bounds, currently, there's something going wrong with the singleton pattern instance I've created. So I'll get back to you.

**Reply by SuperSodaSea** [reply]
Maybe you need to set its [`hitArea`](https://pixijs.download/dev/docs/PIXI.DisplayObject.html#hitArea).

**Reply by LeviSTA** [reply]
Your solution ended up working. Thank you very much! It was just something else in my code that didn't work
